src/constants.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+export const VDB_KIND: string = 'VirtualDatabase';
+export const DDL_FILE_EXT: string = '.ddl';
+export const TEMPLATE_NAME: string = '$!TEMPLATE!$';
+export const DDL_NAME_PLACEHOLDER: string = '$!VDB_NAME_PLACEHOLDER!$';
+export const RELATIONAL_DB_KEY: string = 'RelationalDB';
+export const RELATIONAL_DB_TYPES: string[] = [
```

Comment:
Where does this list come from? Should it come from upstream DV somewhere? How do we keep it in sync with what's supported by the runtime?

Reply:
Currently it is hard coded using https://github.com/teiid/teiid-openshift-examples/blob/master/datasources.adoc as source. In future we will use an automatically generated json file.